google_maps/google_maps.py

```diff
@@ -15,23 +17,25 @@ class GoogleMapsXBlock(XBlock):
     """
     XBlock providing a Google Map to a course.
     """
+    # Class level variables
```

Comment:
For future trainees so they see this and then google about it and for pointing out where the variables should be declared.